dask/dataframe/io/parquet/arrow.py

```diff
@@ -203,22 +203,27 @@ def read_metadata(
 
         if gather_statistics:
             stats = []
+            skip_cols = set()  # Columns with min/max = None detected
             for row_group in row_groups:
                 s = {"num-rows": row_group.num_rows, "columns": []}
                 for i, name in enumerate(names):
-                    column = row_group.column(i)
-                    d = {"name": name}
-                    if column.statistics:
-                        cs_min = column.statistics.min
-                        cs_max = column.statistics.max
-                        d.update(
-                            {
-                                "min": cs_min,
-                                "max": cs_max,
-                                "null_count": column.statistics.null_count,
-                            }
-                        )
-                    s["columns"].append(d)
+                    if name not in skip_cols:
+                        column = row_group.column(i)
+                        d = {"name": name}
+                        if column.statistics:
+                            cs_min = column.statistics.min
+                            cs_max = column.statistics.max
+                            if None in [cs_min, cs_max]:
+                                skip_cols.add(name)
+                                continue
```

Comment:
Perfect. This should probably be done in fastparquet too when generating the stats values, but I suppose in principle, people might want to use them for something other than making dask divisions lists.